helmfile/charts/networkpolicy-service/values.yaml

```
@@ -94,6 +94,8 @@ monitoring:
         - 443
 opensearch:
   enabled: true
+  client:
+    enabled: false
```

Oops, must've slipped through 

Could you also add
```suggestion
  client:
    enabled: false
  data:
    enabled: false
```

---

but i dont use, data.enabled anywhere 

do you want me to use it ?